.evergreen/publish.sh

```diff
@@ -34,4 +34,5 @@ SYSTEM_PROPERTIES="-Dorg.gradle.internal.publish.checksums.insecure=true -Dorg.g
 
 ./gradlew -version
 ./gradlew ${SYSTEM_PROPERTIES} --stacktrace --info  ${TASK}
-./gradlew ${SYSTEM_PROPERTIES} --stacktrace --info :bson-scala:${TASK} :driver-scala:${TASK} -PdefaultScalaVersions=2.11.12,2.12.10
```

Comment:
I had to split out builds / maybe a change in gradle when we last upgraded, that the multiversion plugin didnt handle.

Reply:
Acknowledged, but kind of weird since I thought the point of this plugin was to allow you to do it the way it is now.

Reply:
Yeah, seems like a bug. Ideally I'd like to know enough gradle to write a stable multi version plugin that allows all versions to be published in one shot, as well as allowing different source routes.